data/provider/src/test/java/tech/pegasys/artemis/provider/JsonProviderTest.java

```diff
@@ -13,19 +13,60 @@
 
 package tech.pegasys.artemis.provider;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
 import com.google.common.primitives.UnsignedLong;
+import java.util.List;
+import org.apache.tuweni.bytes.Bytes32;
 import org.junit.jupiter.api.Test;
 import tech.pegasys.artemis.datastructures.state.BeaconState;
 import tech.pegasys.artemis.datastructures.util.DataStructureUtil;
+import tech.pegasys.artemis.util.SSZTypes.SSZVector;
 
 class JsonProviderTest {
+  JsonProvider jsonProvider = new JsonProvider();
```

Comment:
private final

Reply:
that'll be in a lot of places, i'll fix them.